BuildScript/BuildTasks/TestConfigurationsFactory.cs

```diff
@@ -88,25 +88,26 @@ private void CheckNoDuplicateItemsExist (IReadOnlyCollection<string> configurati
       var configurationID = GetConfigurationID (configurationItems);
       var testAssemblyFileName = Path.GetFileName (testAssemblyFullPath);
       var targetRuntimeInitialFormat = TargetRuntimeConverter.ToTargetFrameworkMoniker (targetRuntime);
-      var executionRuntimeKey = _supportedExecutionRuntimes.GetKeys (executionRuntime).First();
       var excludeCategories = GetExcludeCategories (configurationItems);
+      var excludeCategoriesIDText = excludeCategories != "" ? excludeCategories : "None";
```

Comment:
should we use string.IsNullOrEmpty(excludeCategories) ? Would this be easier to read / more idomatic?